CONTRIBUTING.md

```diff
@@ -287,6 +287,8 @@ The types of rules that are allowed at each paranoia level are as follows:
 * False negatives should not happen at this level
 * Check everything against RFCs and allow listed values for the most popular elements
 
+Each rule which placed into a paranoia level must contain tag `paranoia-level/N`, where N is the PL value, but this tag can only be added if rule does **not** have `nolog` action.
```

Comment:
In my opinion, this is important enough to go at the top of this section to make sure it isn't missed. Maybe as a separate paragraph above “The types of rules that are allowed…”.

A slightly modified suggestion:

Each rule that is placed into a paranoia level must contain the tag `paranoia-level/N`, where _N_ is the PL value, however this tag can only be added if the rule does **not** use the `nolog` action.

Reply:
Thanks, fixed in [b16e6f4](https://github.com/coreruleset/coreruleset/pull/2832/commits/b16e6f417f567eae2d50ee01091c580bd9fc7a67).